browser/brave_profile_prefs.cc

```diff
@@ -33,7 +33,9 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
       registry);
 
   // appearance
+#if !defined(OS_ANDROID)
   BraveThemeService::RegisterProfilePrefs(registry);
```

Comment:
Why only this is in non-android? I think `kLocationBarIsWide` is not related with android?
Or this is just for avoiding compile error?

Reply:
I don't believe the theme service exists on android and this fixed either compile or linking error